### ISSUE TITLE
No Onscreen keyboard installed but setting is shown

### ISSUE THREAD

**Issue by arshubham** [elementary/switchboard-plug-a11y#34]
![image](https://user-images.githubusercontent.com/14235191/42373202-e0565f4a-8131-11e8-8529-761bb6fe6673.png)
No Onscreen keyboard is installed as default, however settings is available with no indication of that. Further the On-screen keyboard setting link gives no indication of any form of what has happened if the keyboard is not installed. It leaves user to wonder that something is wrong.

**Comment by codygarver** [comment]
Fixed in ba4c654

**Comment by arshubham** [comment]
@codygarver you maybe want to revert this for now on juno. Onboard starts fine and works correctly. But on closing in normally it crashes gala and cerbere and I am thrown back to greeter. And after login I have no wingpanel and plank which tells me cerbere is not back up.  I can use the terminal to launch both wingpanel and plank. Only a reboot fixes this or `pkill onboard` to temporarily as a workaround to save current session but I get black block surrounding the windows open momentarily. I will open an issue with gala too.

**Comment by codygarver** [comment]
Rolled back the commit a few mins after it was made based on earlier comment.

This issue is now blocked by crash report at elementary/gala#293

**Comment by davidmhewitt** [comment]
A fix for this crash has now been merged into gala, marking as unblocked.

**Comment by codygarver** [comment]
Fixed in iso 20180910
